docker-compose.staging.yml

```diff
@@ -0,0 +1,322 @@
+version: '3.7'
+
+#### docker-compose up -d;
+## Staging Environment - Used for remote (non-local / cloud) hosting of a Staging site. Code here is almost finished but viewed privately prior to pushing to public Production.
+## Updated 2019-08 - Release 1.2.0 (@ 1.2.0)
+
+services:
+  isle-portainer: ## Renamed to avoid conflicts on systems/servers with portainer already running.
+    image: portainer/portainer
+    container_name: isle-portainer-${CONTAINER_SHORT_ID}
+    command: -H unix:///var/run/docker.sock --no-auth
+    env_file:
+      - demo.env
```

Comment:
Is this the correct env file here, or should it be local.env?

Reply:
Well spotted. This is a typo introduced from a recent diff, apologies. I've fixed it in all files and pushed a new commit.